@orelgueta  @VictorBarbosaMartins 

This is still in draft, as test are not working (and I am not sure what is going wrong):

```
docker run --rm -it -v "$(pwd)/external:/workdir/external" simtools-dev bash -c "$(cat ./entrypoint.sh) && bash"

 pytest --no-cov
ImportError while loading conftest '/workdir/external/simtools/tests/conftest.py'.
tests/conftest.py:12: in <module>
    from simtools.corsika.corsika_output import CorsikaOutput
simtools/corsika/corsika_output.py:11: in <module>
    from corsikaio.subblocks import event_header, get_units_from_fields, run_header
E   ImportError: cannot import name 'get_units_from_fields' from 'corsikaio.subblocks' (/workdir/conda/lib/python3.9/site-packages/corsikaio/subblocks/__init__.py)
```

Suggestions to solve this are welcome!

--- a/dev/Dockerfile
+++ b/dev/Dockerfile
@@ -13,9 +13,7 @@ RUN apt-get update && apt-get install -y \
     git \
     libgsl-dev \
     make \
-    unzip \
-    vim \
-    wget && \
+    unzip && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # corsika and sim_telarray
@@ -27,37 +25,34 @@ RUN cd sim_telarray && \
     find . -name "*.tar.gz" -exec rm -f {} \; && \
     cd ..
 
-RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$(uname -p).sh -O ~/miniconda.sh && \
-    /bin/bash ~/miniconda.sh -b -p /workdir/conda && \
-    rm ~/miniconda.sh && \
-    /workdir/conda/bin/conda clean -tipy
-
-RUN wget --quiet https://raw.githubusercontent.com/gammasim/simtools/main/environment.yml
-
 From ubuntu:22.10
 WORKDIR /workdir
 ENV DEBIAN_FRONTEND=noninteractive
 COPY --from=build_image /workdir/sim_telarray/ /workdir/sim_telarray/
-COPY --from=build_image /workdir/conda /workdir/conda
-COPY --from=build_image /workdir/environment.yml /workdir/environment.yml
 
 RUN apt-get update && apt-get install -y \
+    bc \
     bzip2 \
     gfortran \
     libgsl-dev \
-    bc \
-    unzip && \
+    python3-pip \
+    unzip \
+    wget && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
-ENV PATH /workdir/conda/bin:$PATH
+RUN wget --quiet https://bootstrap.pypa.io/get-pip.py && \
+    python3 get-pip.py && \
+    ln -s /usr/bin/python3 /usr/bin/python
 
-RUN conda env update -n base --file environment.yml && \
-    conda clean --all
+RUN wget --quiet https://raw.githubusercontent.com/gammasim/simtools/main/setup.py && \
+    wget --quiet https://raw.githubusercontent.com/gammasim/simtools/main/pyproject.toml
+
+RUN cd /workdir/ && \
+    pip install '.[tests,dev,doc]'
 
 ENV SIMTEL_PATH="/workdir/sim_telarray/"
 SHELL ["/bin/bash", "-c"]
 
-RUN source /root/.bashrc && \
-    conda init bash
+RUN source /root/.bashrc
 
 WORKDIR /workdir/external

--- a/dev/Dockerfile
+++ b/dev/Dockerfile
@@ -32,7 +32,7 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-$(u
     rm ~/miniconda.sh && \
     /workdir/conda/bin/conda clean -tipy
 
-RUN wget --quiet https://raw.githubusercontent.com/gammasim/simtools/master/environment.yml
+RUN wget --quiet https://raw.githubusercontent.com/gammasim/simtools/main/environment.yml
 
 From ubuntu:22.10
 WORKDIR /workdir

--- a/dev/Dockerfile
+++ b/dev/Dockerfile
@@ -20,7 +20,7 @@ RUN apt-get update && apt-get install -y \
 RUN mkdir sim_telarray
 COPY corsika7.7_simtelarray.tar.gz sim_telarray
 RUN cd sim_telarray && \
-    tar -xvzf corsika7.7_simtelarray.tar.gz && \
+    tar -xvf corsika7.7_simtelarray.tar.gz && \
     ./build_all prod5 qgs2 gsl && \
     find . -name "*.tar.gz" -exec rm -f {} \; && \
     cd ..
@@ -53,6 +53,5 @@ RUN cd /workdir/ && \
 ENV SIMTEL_PATH="/workdir/sim_telarray/"
 SHELL ["/bin/bash", "-c"]
 
-RUN source /root/.bashrc
 
 WORKDIR /workdir/external

